### ISSUE TITLE
make it possible to use a package that replaces cakephp/cakephp

### ISSUE THREAD

**Issue by stosef** [the-kbA-team/phinx-cake2#17]
The composer package "kba-team/cakephp" is used as a replacement for "cakephp/cakephp". This is done with using the "replace" section of the compose.json config. See here: https://getcomposer.org/doc/04-schema.md#replace

In order for this to work, the package name that replaces "cakephp/cakephp" has to be found. Reason is that I do not want to use "kba-team/cakephp" hardcoded, I want to use the features of composer itself to find out if "cakephp/cakephp" is replaced and by which package.

Basically I want to replace

```php
$pathCakeInVendorDir = $vendorDir . DS . 'cakephp' . DS . 'cakephp' . DS . 'lib';
```
with something like

```php
// getVendorPath takes care of eventually installed replacement package
$pathCakeInVendor = getVendorPath('cakephp/cakephp'). DS . 'lib';
```
